### DISCUSSION TITLE
How to access PageSelector when PDF file is viewed

### DISCUSSION THREAD

**Discussion by MarekPio** [MicrosoftEdge/WebView2Feedback#4072] [Q&A]
Hello,
I want to use WebView2 to view PDF file and detect when the user reaches the last page of the file.
How to access:
- current page number,
- last page number.

**Comment by victorhuangwq** [comment]
@monica-ch could you take a look at this if possible?

**Comment by monica-ch** [comment]
@MarekPio Thanks for reaching out and right now we don't have an equivalent API for this, but we can take it as a feature request. Would you like to share on why you need this in your app and what kind of app is this?
